src/clusterer.rs

```diff
@@ -12,14 +13,12 @@ use rayon::prelude::*;
 
 /// Given a list of genomes, return them clustered. Use dashing for first pass
 /// analysis, then fastani as the actual threshold.
-pub fn cluster<P: PreclusterDistanceFinder>(
+pub fn cluster<P: PreclusterDistanceFinder, C: ClusterDistanceFinder + std::marker::Sync>(
     genomes: &[&str],
     preclusterer: &P,
-    fastani_threshold: f32,
-    fastani_min_aligned_threshold: f32,
-    fastani_fraglen: u32,
+    clusterer: &C,
 ) -> Vec<Vec<usize>> {
-    assert!(fastani_threshold > 1.0);
+    // assert!(fastani_threshold > 1.0);
```

Comment:
@wwood I'm not sure where this assertion should go now. Within an initialiser/verifier clusterer method?

Reply:
initialiser I guess, which would be called outside this method I think